### ISSUE TITLE
Not seeing buttons on JupyterLab

### ISSUE THREAD

**Issue by dhruvbalwada** [pangeo-data/pangeo-cloud-federation#448]
I just noticed that the icons and text for a lot of the buttons on JupyterLab on ocean.pangeo.io are not visible. Note the blank line next to where the drop down menu for code/markdown/raw is present
<img width="863" alt="Screen Shot 2019-10-14 at 7 58 51 PM" src="https://user-images.githubusercontent.com/18236610/66796628-1cbeb280-eebd-11e9-8333-3c04029b29f5.png">

The buttons are still there because I can click the empty white space and things happen, but I don't know what I am clicking.

**Comment by jhamman** [comment]
I'm not seeing this. If you refresh your session or log out and log back in, do you still see the same missing features? What about if you are in a different browser?

**Comment by dhruvbalwada** [comment]
I seem to only be having the problem in Chrome, but works fine in safari. Logging in and out does not seem to fix it.

**Comment by rabernat** [comment]
I am seeing this in Chrome too:

![image](https://user-images.githubusercontent.com/1197350/66834728-321ef580-ef2c-11e9-9359-5546be5b0005.png)

**Comment by rabernat** [comment]
The only errors I am seeing in my javascript console are
```
dask-logo.svg:1 GET https://ocean.pangeo.io/user/0000-0001-5999-4917/proxy/8787/statics/images/dask-logo.svg?1571145967857 500
```

**Comment by rabernat** [comment]
More detailed errors

```
dask-logo.svg:1 GET https://ocean.pangeo.io/user/0000-0001-5999-4917/proxy/8787/statics/images/dask-logo.svg?1571145963762 500
Image (async)
(anonymous) @ dashboard.js:418
testDaskDashboard @ dashboard.js:397
(anonymous) @ dashboard.js:296
(anonymous) @ dashboard.js:7
push.QJ1H.__awaiter @ dashboard.js:3
factory @ dashboard.js:290
_execute @ poll.js:242
execute @ poll.js:192
setTimeout (async)
schedule @ poll.js:199
async function (async)
schedule @ poll.js:186
(anonymous) @ poll.js:247
Promise.then (async)
_execute @ poll.js:243
execute @ poll.js:192
requestAnimationFrame (async)
schedule @ poll.js:196
async function (async)
schedule @ poll.js:186
start @ poll.js:207
(anonymous) @ poll.js:49
requestAnimationFrame (async)
Poll @ poll.js:49
_startUrlCheckTimer @ dashboard.js:289
URLInput @ dashboard.js:180
DaskDashboardLauncher @ dashboard.js:108
DaskSidebar @ sidebar.js:18
activate @ index.js:97
(anonymous) @ index.js:147
Promise.then (async)
push.2R+v.Application.activatePlugin @ index.js:146
(anonymous) @ index.js:266
push.2R+v.Application.start @ index.js:265
main @ index.out.js:1518
load (async)
ANye @ index.out.js:1564
__webpack_require__ @ bootstrap:84
0 @ bootstrap:221
__webpack_require__ @ bootstrap:84
checkDeferredModules @ bootstrap:45
webpackJsonpCallback @ bootstrap:32
(anonymous) @ vendors~main.1273ed9bc3508629de5e.js:1
```

And also 

```
TeX-AMS-MML_HTMLorMML-full.js?V=2.7.6:70 GET about:blank net::ERR_UNKNOWN_URL_SCHEME

  getComparisonWidths | @ | TeX-AMS-MML_HTMLorMML-full.js?V=2.7.6:70
  | testFont | @ | TeX-AMS-MML_HTMLorMML-full.js?V=2.7.6:70
  | checkWebFont | @ | TeX-AMS-MML_HTMLorMML-full.js?V=2.7.6:70
  | execute | @ | MathJax.js?config=Te…Until=configured:19
  | cb | @ | MathJax.js?config=Te…Until=configured:19
  | start | @ | MathJax.js?config=Te…Until=configured:19
  | loadWebFont | @ | TeX-AMS-MML_HTMLorMML-full.js?V=2.7.6:70
  | loadWebFont | @ | TeX-AMS-MML_HTMLorMML-full.js?V=2.7.6:70
  | lookupChar | @ | TeX-AMS-MML_HTMLorMML-full.js?V=2.7.6:70
  | handleVariant | @ | TeX-AMS-MML_HTMLorMML-full.js?V=2.7.6:70
  | HTMLhandleVariant | @ | TeX-AMS-MML_HTMLorMML-full.js?V=2.7.6:70
  | toHTML | @ | TeX-AMS-MML_HTMLorMML-full.js?V=2.7.6:70
  | toHTML | @ | TeX-AMS-MML_HTMLorMML-full.js?V=2.7.6:70
  | toHTML | @ | TeX-AMS-MML_HTMLorMML-full.js?V=2.7.6:70
  | toHTML | @ | TeX-AMS-MML_HTMLorMML-full.js?V=2.7.6:70
  | toHTML | @ | TeX-AMS-MML_HTMLorMML-full.js?V=2.7.6:63
  | Translate | @ | TeX-AMS-MML_HTMLorMML-full.js?V=2.7.6:70
  | processOutput | @ | MathJax.js?config=Te…Until=configured:19
  | execute | @ | MathJax.js?config=Te…Until=configured:19
  | WAITEXECUTE | @ | MathJax.js?config=Te…Until=configured:19
  | cb | @ | MathJax.js?config=Te…Until=configured:19
  | WAITEXECUTE | @ | MathJax.js?config=Te…Until=configured:19
  | cb | @ | MathJax.js?config=Te…Until=configured:19
  | Process | @ | MathJax.js?config=Te…Until=configured:19
  | call | @ | MathJax.js?config=Te…Until=configured:19
  | WAITEXECUTE | @ | MathJax.js?config=Te…Until=configured:19
  | cb | @ | MathJax.js?config=Te…Until=configured:19
  | Execute | @ | MathJax.js?config=Te…Until=configured:19
  | loadComplete | @ | MathJax.js?config=Te…Until=configured:19
  | execute | @ | MathJax.js?config=Te…Until=configured:19
  | setTimeout (async) |   |  
  | checkLength | @ | MathJax.js?config=Te…Until=configured:19
  | execute | @ | MathJax.js?config=Te…Until=configured:19
  | cb | @ | MathJax.js?config=Te…Until=configured:19
  | setTimeout (async) |   |  
  | start | @ | MathJax.js?config=Te…Until=configured:19
  | create | @ | MathJax.js?config=Te…Until=configured:19
  | Styles | @ | MathJax.js?config=Te…Until=configured:19
  | initFont | @ | TeX-AMS-MML_HTMLorMML-full.js?V=2.7.6:70
  | execute | @ | MathJax.js?config=Te…Until=configured:19
  | cb | @ | MathJax.js?config=Te…Until=configured:19
  | Process | @ | MathJax.js?config=Te…Until=configured:19
  | Push | @ | MathJax.js?config=Te…Until=configured:19
  | Init | @ | MathJax.js?config=Te…Until=configured:19
  | Init | @ | MathJax.js?config=Te…Until=configured:19
  | (anonymous) | @ | MathJax.js?config=Te…Until=configured:19
  | (anonymous) | @ | Main.js?V=2.7.6:19


```

**Comment by jhamman** [comment]
> The only errors I am seeing in my javascript console are
> 
> ```
> dask-logo.svg:1 GET https://ocean.pangeo.io/user/0000-0001-5999-4917/proxy/8787/statics/images/dask-logo.svg?1571145967857 500
> ```

I don't think this is an error. I think this is just how dask-labextension checks for an active dashboard.

**Comment by rabernat** [comment]
Relevant issue: https://github.com/jupyterlab/jupyterlab/issues/7129

Recommended fix it to upgrade to jupyterlab v 1.1.1. We are currently on 1.0.9.

Is there anything keeping us pinned at that version?

**Comment by dhruvbalwada** [comment]
As of this second the icons are back, and I don't think I changed anything on my side except restart the server a few times.

**Comment by jhamman** [comment]
@rabernat - once #450 is merged, we'll be up to 1.1.4.
